src/test/mod.rs

```diff
@@ -8,9 +8,15 @@ use reqwest::{Client, Method, RequestBuilder};
 use std::sync::{Arc, Mutex, MutexGuard};
 
 pub(crate) fn wrapper(f: impl FnOnce(&TestEnvironment) -> Result<(), Error>) {
+    use std::panic;
```

Comment:
Nit: can you move the import outside of the function? Or just using `std::panic::...` instead or just `panic::`.